Podfile

```diff
@@ -1,8 +1,10 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
-pod 'IQKeyboardManagerSwift'
 use_frameworks!
 
+pod 'IQKeyboardManagerSwift'
```

Comment:
Seems we are using "IQKeyboardManagerSwift" from CocoaPods and we don't have any specific modifications, that the target 'AppConnectSwift' seek to be using to strictly sticking to. Please let me know if missing anything.